build/build_wheel_docker_entrypoint.sh

```diff
@@ -62,7 +62,9 @@ esac
 
 export JAX_CUDA_VERSION=$3
 python setup.py bdist_wheel --python-tag "$PY_TAG" --plat-name "$PLAT_NAME"
-if python -m auditwheel show dist/jaxlib-*.whl  | grep -v 'platform tag: "manylinux2010_x86_64"' > /dev/null; then
+if ! python -m auditwheel show dist/jaxlib-*.whl  | grep 'platform tag: "manylinux2010_x86_64"' > /dev/null; then
```

Comment:
Please verify the wheel you get out passes the test when done manually!
(I don't really parse the shell here.)

Reply:
I used this to generate the just-released jaxlib 0.1.57 wheels, and the tests pass (both mac and linux).